app/controllers/idv/in_person/ssn_controller.rb

```diff
@@ -0,0 +1,106 @@
+module Idv
+  module InPerson
+    class SsnController < ApplicationController
+      include IdvStepConcern
+      include StepIndicatorConcern
+      include StepUtilitiesConcern
+      include Steps::ThreatMetrixStepHelper
+      include ThreatMetrixConcern
+
+      before_action :renders_404_if_in_person_ssn_info_controller_enabled_flag_not_set
+      before_action :confirm_verify_info_step_needed
+      before_action :confirm_in_person_address_step_complete
+      before_action :confirm_repeat_ssn, only: :show
+      ## TO DO: ARE WE DOING THREATMETRIX? IF YES, KEEP
```

Comment:
I wonder if this should be removed before merging (or if not, how to capture when to remove it in the future).

[This commit on your branch](https://github.com/18F/identity-idp/pull/8744/commits/b6fe069b70101a616baaf3c6ad7a6a0145f6128f) seems to imply we won't need it, but I could be mistaken.

Reply:
@night-jellyfish  I left the comment it as a reminder to check/ask during code review, so thanks for mentioning it.  Inside the FSM ssn controller (app/services/idv/steps/in_person/ssn_step.rb) we do not have threatmetrix methods like Team Ada does in their SSN controller.  We **do** have threatmetrix stuff in our FSM SSN template (app/views/idv/in_person/ssn.html.erb) - so I included in my template. 

So, I think I need to delete the comment but keep `before_action :override_csp_for_threat_metrix_no_fsm`.

Reply:
@night-jellyfish  I remove before_action override_csp_for_threat_metrix_no_fsm as I am not seeing it being used in FSM SSN. (I only see it being used in Doc Auth SSN.) The name also has no_fsm in it so don't think I need it. I cleaned up the tests that went with it also, see commit 04fcd86cb216aa54653b038465e3f7ba25ce24e3